pkg/monitor/cluster/podconditions.go

```diff
@@ -59,6 +59,16 @@ func (mon *Monitor) _emitPodConditions(ps *corev1.PodList) {
 			continue
 		}
 
+		if p.Status.Reason == "Preempting" {
+			for _, cs := range p.Status.ContainerStatuses {
+				if cs.State.Terminated != nil {
+					if cs.State.Terminated.Reason == "Completed" {
+						continue
+					}
+				}
+			}
+		}
+
```

Comment:
Quick idea, what about:

1. extracting the hardcoded strings into constants (it has the advantatge that if the string changes, the line checking the conditions will not change)

2. simplifying the 2 if's into a single one (advantage of less nested code and a bit cleaner)

Something like:
```go
const Preempting = "Preempting"
const Completed = "Completed"

if p.Status.Reason == Preempting {
    for _, cs := range p.Status.ContainerStatuses {
        if cs.State.Terminated != nil && cs.State.Terminated.Reason == Completed {
            continue
        }
    }
}
```